buildah/Dockerfile

```
@@ -0,0 +1,31 @@
+FROM launcher.gcr.io/google/ubuntu16_04
+
+RUN \
+  # Install runc
+  apt-get update && \
+  apt-get -y install wget && \
+  wget -O /bin/runc https://github.com/opencontainers/runc/releases/download/v1.0.0-rc5/runc.amd64 && \
+  chmod +x /bin/runc && \
+  PATH=/usr/local/bin/:$PATH && \
+  runc --help && \
+  # Install buildah dependencies
+  apt-get -y install software-properties-common && \
+  add-apt-repository -y ppa:alexlarsson/flatpak && \
```

Can move these repositories up above line 5 and thereby get away with just one `apt-get update` call.

---

I wanted to separate the build into "install runc" and "install buildah", and fail faster if runc didn't install correctly. I don't think it probably adds much latency to have these split, and I think the conceptual separation is worth the small amount of duplication.

---

Add a comment to that effect?